carina-core/src/main/resources/config.properties

```diff
@@ -67,6 +67,7 @@ auto_download_apps=NULL
 auto_download_folder=NULL
 custom_artifacts_folder=NULL
 scroll_to_element_y_offset=120
+element_loading_strategy=BY_PRESENCE
```

Comment:
to avoid all kind of regression I suggest to organize strategies:
BY_DEFAULT
BY_PRESENCE
BY_VISIBILITY

i.e. by default completely reuse existing approach. 
by_presence and by_visibility - fully new approaches

Reply:
done